node-scripts/yt-description.js

```diff
@@ -311,7 +311,7 @@ Music from Epidemic Sound
 
 This description was auto-generated. If you see a problem, please open an issue: https://github.com/CodingTrain/thecodingtrain.com/issues/new`;
 
-  let filename = data.videoId;
+  let filename = data.videoId || /\/((?:.(?!\/))+)$/.exec(pageURL)[1];
```

Comment:
I would prefer the following:

```
slug-name.txt (no videoID)
slug-name_videoID.txt (if videoID)
```

It's very hard to find and sort files by videoID.